Assets/MRTK/Core/Providers/InputSimulation/Editor/MixedRealityInputSimulationProfileInspector.cs

```diff
@@ -190,9 +203,9 @@ public override void OnInspectorGUI()
                     EditorGUILayout.PropertyField(navigationStartThreshold);
                     EditorGUILayout.Space();
 
-                    EditorGUILayout.PropertyField(defaultControllerDistance);
-                    EditorGUILayout.PropertyField(controllerDepthMultiplier);
-                    EditorGUILayout.PropertyField(controllerJitterAmount);
```

Comment:
jFYI if moving the controller/depth/jitter was intentionally done before all of the hand/motionController sections, that's alright, just going forward changing UI/menu locations can be annoying for people who are consuming  it (i.e. moving locations arbitrarily can be frustrating to use)